### ISSUE TITLE
BioModel Discussion Issue for eos9yy1

### ISSUE THREAD

**Issue by Zainab-ik** [ersilia-os/eos9yy1#19]
**Summary !!!**

- **BioModel Name**: Shah2020 -  Predicting liver cytosol stability of small molecules.

- **BioModel Tag**: fair AIML, Machine learning model

- **Metadata Comments**: 

1. Cytosolic stability is a measure of metabolism. Does metabolism classify as a metadata in this case?
2. Is there any relationship between cytosolic stability and cyp450 enzymes and CYP-mediated metabolism?

- **Contributor**:
> 
 | roles             | Names | 
 | :---------------- | :------: | 
 | Curator        |   @Zainab-ik   | 
 | Code contributor      |   Pauline-Banye       | 
 | Reviewer       |   @GemmaTuron    | 

- **Curation status** :  manually curated

- **Annotation file**: [here](https://docs.google.com/spreadsheets/d/11qODwr1miayNHcUruW0WjGdZJIMQ9YXLDJaodURwFWQ/edit?usp=drive_link)

**Comment by GemmaTuron** [comment]
I agree cytosolic stability falls within metabolism, but I would only label as CYP mediated metabolism the assays that focus on CYPs.
And I would not use RDKit as a label as it is such a well known package that literally all the models use it. If we add it in one, we should add it in all others

**Comment by Zainab-ik** [comment]
> I agree cytosolic stability falls within metabolism, but I would only label as CYP mediated metabolism the assays that focus on CYPs. And I would not use RDKit as a label as it is such a well known package that literally all the models use it. If we add it in one, we should add it in all others
